### ISSUE TITLE
warning in external Lib: LOOP FOR before WITH

### ISSUE THREAD

**Issue by lispm** [shop-planner/shop3#19]
Hmm, in jenkins/ext/fiveam-asdf/code.lisp (I know, an external lib)

In the PERFORM method the LOOP should better have the WITH before the FOR.
It's strictly not necessary, but the vars will be initialized before the FOR var.
LispWorks warns...

```
(defmethod perform ((op test-op) (sys fiveam-tester-system))
  (let* ((test-syms
           (loop with test-name and package-name and test-sym and package
                 for x in (test-names sys)
```

**Comment by lispm** [comment]
similar in common/state-utils.lisp in method state-trajectory

**Comment by lispm** [comment]
similar in io/input.lisp

DEFGENERIC DOMAIN-INCLUDE-PARSE

**Comment by rpgoldman** [comment]
The SHOP3 loops are fixed in dcb67364d6da59133224cc850a057f7e5e173f48

Fixing fiveam-asdf separately.
